### PR TITLE
Concourse: Add miscellaneous concourse atc flags.

### PR DESCRIPTION
**Pull request by jmcarp**
**What this PR does / why we need it**:

Adds a few missing configuration options to the concourse atc, mostly useful to us for compliance purposes.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

@frodenas @william-tran

### CODE REVIEW

**Review by william-tran — COMMENTED** [review]
Thanks @jmcarp, I'd like to hold off on an update to 3.11 until this issue gets sorted out: https://github.com/concourse/concourse/issues/2165#issuecomment-382869258

**Comment by brianmartin** [comment]
Hey @jmcarp, these changes look quite useful to us as well!

Looks like 3.12 has been released and resolves the memory leak concern. Moreover, 3.13 is out as of yesterday.

**Comment by william-tran** [comment]
I'd be good to go with 3.13 in this PR, let us know if you want to take that on or have those new features added in another PR. I see:
- `The ATC can now be configured with a global default build_logs_to_retain`
- `This requires a new port to be reachable on the worker node from the web node: 7799`
- `Added a new authentication provider for teams using OpenID Connect (OIDC)`

**Comment by jmcarp** [comment]
@william-tran: I just bumped the image tag to `3.13.0`, exposed `7799` on workers, and added build log retention options.

**Comment by brianmartin** [comment]
@jmcarp Looks good to me! I do see a few more flags added for 3.13 related to Datadog metrics (see: https://github.com/concourse/atc/pull/269#issue-183163123). No worries if you want to keep this PR more tightly scoped though – I can add those a little later if they don't make sense in this PR.

**Comment by jmcarp** [comment]
@brianmartin it looks like we're missing a bunch of options around metrics, so maybe we can add them all together in a separate patch.

**Comment by brianmartin** [comment]
@jmcarp Sounds totally reasonable.

**Comment by william-tran** [comment]
@jmcarp could you minor bump the chart to 1.5.0? We have a 1.4.0 about to be merged.

**Comment by jmcarp** [comment]
Bumped to 1.5.0 @william-tran

File: stable/concourse/values.yaml [hunk 1]
```diff
@@ -279,6 +279,37 @@ web:
       enabled: false
       port: 9391
 
+  ## Set secure flag on auth cookies
+  ##
+  cookieSecure: false
```
Comment (william-tran) [inline stable/concourse/values.yaml:284]: Is this a sensible default?

File: stable/concourse/values.yaml [hunk 2]
```diff
@@ -279,6 +279,37 @@ web:
...
+  ## Length of time for a intercepted session to be idle before terminating, in Go duration format
+  ##
+  interceptIdleTimeout: ""
+
+  ## Logging configuration
+  logging:
+
+    ## The log level for the ATC
+    ##
+    level: ""
```
Comment (william-tran) [inline stable/concourse/values.yaml:299]: what does blank mean here?

**Review by william-tran — CHANGES_REQUESTED** [review]
We also need to add entries to the values table in the README

**Comment by william-tran** [comment]
With generic env vars support via #6441, I will close this PR, as you can use that instead. There are so many options that can be enabled via env vars alone. To keep the chart from getting too complicated with features, I won't take any more PRs that are a straight value to env var / command line arg thing.

**Pull request closed without merging**